src/components/organisms/AddOffer.vue

```diff
@@ -0,0 +1,374 @@
+<template>
```

Comment:
Change the z-index of the mapPreview here or in the DefaultError, because now the mapPreview is higher